resources/giwscripts/giwtype.py

```diff
@@ -30,10 +30,16 @@ def get_buffer_metadata(obj_name, picked_obj, debugger_bridge):
 
     width = int(picked_obj['cols'])
     height = int(picked_obj['rows'])
+    if hasattr(picked_obj,'flags'):
     flags = int(picked_obj['flags'])
+    else:
+        flags = int(picked_obj['type'])
 
     channels = ((((flags) & CV_MAT_CN_MASK) >> CV_CN_SHIFT) + 1)
+    if hasattr(picked_obj['step'],'buf'):
     row_stride = int(int(picked_obj['step']['buf'][0])/channels)
```

Comment:
I'm not sure github is correctly showing the diff, but shouldn't this line be indented to its if block?

Reply:
of course you are right, line 40 has to be indented, I must have made some mistake in merging this solution to the github branch.